from ai_highscore.custom_exceptions.configuiration_collection_merge_error import ConfigurationCollectionMergeError, \
    ConfigurationCollectionMergeErrorType
